packages/bot-skeleton/src/scratch/hooks/data_category.js

```diff
@@ -78,11 +78,10 @@ Blockly.DataCategory.search = function (variableModelList) {
 Blockly.DataCategory.addCreateButton = function (xmlList, workspace) {
     const buttonXml = goog.dom.createDom('button');
     // Set default msg, callbackKey, and callback values for type 'VARIABLE'
-    const msg = localize('Create variable');
+    const msg = localize('Create');
     const callbackKey = 'CREATE_VARIABLE';
-    const callback = function (button) {
-        const buttonWorkspace = button.getTargetWorkspace();
-        Blockly.Variables.createVariable(buttonWorkspace, null, '');
+    const callback = function (variableName) {
+        Blockly.derivWorkspace.createVariable(variableName, '');
```

Comment:
Try to refrain from using the global `Blockly.derivWorkspace` if you can access the workspace through passed args, i.e. through `button.getTargetWorkspace()` as it was in the previous version of this file. Was there anything wrong with the original implementation (other than the string being different)?